source/core/parameters.cc

```diff
@@ -1661,15 +1636,20 @@ namespace Parameters
         prm.get_bool("expand particle-wall contact search");
       target_size = prm.get_double("target size");
 
-
-      translate = prm.get_bool("translate");
-      delta_x   = prm.get_double("delta_x");
-      delta_y   = prm.get_double("delta_y");
-      delta_z   = prm.get_double("delta_z");
-
-      rotate = prm.get_bool("rotate");
-      axis   = prm.get_integer("axis");
-      angle  = prm.get_double("angle");
+      // Initial translation
+      std::string              translation_str = prm.get("initial translation");
+      std::vector<std::string> translate_str_list =
+        Utilities::split_string_list(translation_str);
+      translate = Utilities::string_to_double(translate_str_list);
```

Comment:
Why a vector of double and not a Tensor as done for axis?

Reply:
I did the change, but GridTools::shift need to use a Point<spacedim> in our case... 

in short, it doesn't really matter if we are using a Tensor<1,3> or a std::vector for translation variable since we need to create a Point<spacedim> in the GridTools::shift input using the component that Tensor/std::vector  --> translation variable is being used to store the info

Reply:
I did try to use a Tensor<1, spacedim> instead  of Point<spacedim> in GridTools::shift() , but it wasn't working...

Reply:
You can create a point from a tensor. That's the second type of constructor for the Point Class: dealii.org/current/doxygen/deal.II/classPoint.html